operator/cmd/mesh/testdata/manifest-generate/input/istio_csr.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+metadata:
+  namespace: istio-system
+spec:
+  meshConfig:
+    trustDomain: cluster.local
+  values:
+    global:
+      caAddress: cert-manager-istio-csr.cert-manager.svc:443
+    pilot:
+      env:
+        ENABLE_CA_SERVER: false
+      extraContainerArgs:
+        - --tlsCertFile=/etc/cert-manager/tls/tls.crt
+        - --tlsKeyFile=/etc/cert-manager/tls/tls.key
+        - --caCertFile=/etc/cert-manager/ca/root-cert.pem
+      volumeMounts:
+        - name: cert-manager
```

Comment:
Doesn't the base chart already have the volumes for this (with optional mount) ? 

I'm confused on what is the purpose for this file - nothing should need to be configured if the defaults are used.